docs/api/usage.rst

```diff
@@ -86,15 +86,15 @@ may be used:
         narrowed = Concrete.cast_as_concrete(model)
         reveal_type(narrowed) # Concrete1 | Concrete2 | Concrete3
 
-    def takes_model_cls(model: type[AbstractModel]) -> None:
-        narrowed = Concrete.cast_as_concrete(model)
+    def takes_model_cls(model_cls: type[AbstractModel]) -> None:
+        narrowed = Concrete.cast_as_concrete(model_cls)
         reveal_type(narrowed) # type[Concrete1] | type[Concrete2] | type[Concrete3]
```

Comment:
> Clarify examples that are using type[] that they take a model class not a model instance.

This could be seen as ambiguous as the example immediately above is using `model` as an instance.